cob_moveit_bringup/package.xml

```diff
@@ -22,5 +22,7 @@
   <exec_depend>moveit_setup_assistant</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>tf</exec_depend>
+  <exec_depend>IndustrialFCL</exec_depend>
```

Comment:
IndustrialFCL is part of `industrial_collision_detection` which is not released...
what do you use it for?